.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+      - name: Update Chart READMEs
+        uses: addnab/docker-run-action@v3
+        with:
+          image: jnorwood/helm-docs:v1.10.0
+          options: -v ${{ github.workspace }}/charts:/helm-docs
+          run: |
+            helm-docs --log-level debug
+      - name: Commit manifest files
+        id: make-commit
+        run: |
+          git add gradle.properties $(find charts -name Chart.yaml) $(find charts -name README.md)
+          git commit --message "Prepare release ${{ needs.release-version.outputs.RELEASE_VERSION }}"
+
+          echo "commit=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
+
+  # Release: Maven Artifacts
+  maven-release:
+    name: Publish extension's release version to maven repository
+    needs: [ prepare-hotfix-release, release-version ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    if: needs.release-version.outputs.RELEASE_VERSION
+    steps:
+      - name: Export RELEASE_VERSION env
+        run: |
+          echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
+
+      # Set-Up
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-java
+
+      # Import GPG Key
+      - uses: ./.github/actions/import-gpg-key
+        name: "Import GPG Key"
+        with:
+          gpg-private-key: ${{ secrets.ORG_GPG_PRIVATE_KEY }}
+
+      # publish releases
+      - name: Publish version
+        env:
+          OSSRH_PASSWORD: ${{ secrets.ORG_OSSRH_PASSWORD }}
+          OSSRH_USER: ${{ secrets.ORG_OSSRH_USERNAME }}
+        run: |-
+          echo "Publishing Version $RELEASE_VERSION to Sonatype/MavenCentral"
+          ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository --no-parallel -Pversion=$RELEASE_VERSION -Psigning.gnupg.executable=gpg -Psigning.gnupg.passphrase="${{ secrets.ORG_GPG_PASSPHRASE }}"
+
+  docker-release:
+    name: Publish Docker images
+    runs-on: ubuntu-latest
+    needs: [ prepare-hotfix-release, release-version ]
+    permissions:
+      contents: write
+    if: needs.release-version.outputs.RELEASE_VERSION
+
+    strategy:
+      fail-fast: false
+      matrix:
+        variant: [ { dir: edc-controlplane, img: edc-runtime-memory },
+                   { dir: edc-controlplane, img: edc-controlplane-postgresql-hashicorp-vault },
+                   { dir: edc-controlplane, img: edc-controlplane-postgresql-azure-vault },
+                   { dir: edc-dataplane,    img: edc-dataplane-azure-vault },
+                   { dir: edc-dataplane,    img: edc-dataplane-hashicorp-vault },
+                   { dir: edc-tests/runtime, img: mock-connector }]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Export RELEASE_VERSION env
+        run: |
+          echo "RELEASE_VERSION=${{ needs.release-version.outputs.RELEASE_VERSION }}" >> $GITHUB_ENV
```

Comment:
is this variable needed? you could just use `${{ needs.release-version.output.RELEASE_VERSION }}` in line 167

Reply:
Addressed in [c240841](https://github.com/eclipse-tractusx/tractusx-edc/pull/1511/commits/c24084146ae9dad6a139561ae477c8eef49b2764)